src/pages/api/nostr/post-event.ts

```diff
@@ -41,10 +41,10 @@ const parseRequestBody = (body: string) => {
     console.log("Missing or invalid property: publicKey");
     throw new Error('Invalid request data: missing or invalid property');
   }
-  if (!parsedBody.privkey || typeof parsedBody.privkey !== 'string') { 
-    console.log("Missing or invalid property: privateKey");
-    throw new Error('Invalid request data: missing or invalid property');
-  }
+  // if (!parsedBody.privkey || typeof parsedBody.privkey !== 'string') { 
```

Comment:
Some requests won't have a private key if they are signed by the extension. AKA a delete event that I just implemented

Reply:
This will only be accessed if the user signs in with a private key so we should keep it.

Reply:
This throws an error when a user that signs in with an extension trys to post an event because the event won't contain his private key. Thus I removed it as the original logic is expecting everyone to have a private key in the request.